src/libpriv/rpmostree-refts.cxx

```diff
@@ -114,6 +125,26 @@ RpmTs::package_meta (const rust::Str name) const
           retval->_size = headerGetNumber (h, RPMTAG_LONGARCHIVESIZE);
           retval->_buildtime = headerGetNumber (h, RPMTAG_BUILDTIME);
           retval->_src_pkg = headerGetString (h, RPMTAG_SOURCERPM);
+
+          // Get the changelogs
+          struct rpmtd_s nchanges_date_s;
+          _cleanup_rpmtddata_ rpmtd nchanges_date = NULL;
+          nchanges_date = &nchanges_date_s;
+          headerGet (h, RPMTAG_CHANGELOGTIME, nchanges_date, HEADERGET_MINMEM);
+          int ncnum = rpmtdCount (nchanges_date);
+          if (!ncnum)
+            ncnum = 0;
+
+          rust::Vec<uint64_t> epochs;
+          while (ncnum > 0)
+            {
+              uint64_t nchange_date = 0;
+              rpmtdNext (nchanges_date);
+              nchange_date = rpmtdGetNumber (nchanges_date);
+              epochs.push_back (nchange_date);
+              --ncnum;
+            }
+          retval->_changelogs = epochs;
```

Comment:
One thing that trips me up here...I *think* we may want this to be `retval->_changelogs = std::move(epochs);`?